examples/inference_server/inference.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
Can we have a more realistic example in addition to (or instead of) this one? As pointed out in `server.py`, we need to send model weights and inputs to the cluster and get the prediction outputs from the cluster. Why don't we show such a complete example here?

Reply:
That makes sense! I went with a simpler example because I don't have much experience with ML and inference - what do you think would be a good library and model to implement here?